### PR TITLE
Update dependencies to angular v15.2.8

### PR DESCRIPTION
**Pull request by jzolnowski**
Update Angular and related dependencies to v15.2.8 to fix https://github.com/xlts-dev/angularjs-angular-hybrid-angular-cli/pull/8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,18 +50,18 @@
...
+    "@angular-eslint/eslint-plugin": "15.2.1",
+    "@angular-eslint/eslint-plugin-template": "15.2.1",
+    "@angular-eslint/schematics": "15.2.1",
+    "@angular-eslint/template-parser": "15.2.1",
+    "@angular/cli": "15.2.7",
+    "@angular/compiler-cli": "15.2.8",
+    "@angular/language-service": "15.2.8",
     "@playwright/test": "^1.30.0",
-    "@types/angular": "^1.8.4",
+    "@types/angular": "^1.8.5",
```
Comment (gkalpak) [inline package.json:64]: This change is unrelated (i.e. it has nothing to do with the Angular update).

**Review by Splaktar — APPROVED** [review]
